source/versioning/index.html.md.erb

```diff
@@ -7,7 +7,11 @@ weight: 190
 
 # Stay up to date
 
-The GOV.UK Pay API follows the [Semantic Versioning 2.0.0 system](https://semver.org/).
+The GOV.UK Pay API follows the Central Digital and Data Office's [API data and technical standards](https://www.gov.uk/guidance/gds-api-technical-and-data-standards#when-iterating-your-api).
```

Comment:
Change link to `https://www.gov.uk/guidance/gds-api-technical-and-data-standards` as it follows the entirety of the standard.